Buildings/Applications/DataCenters/ChillerCooled/Equipment/BaseClasses/PartialPlantParallel.mo

```diff
@@ -53,14 +49,14 @@ partial model PartialPlantParallel
     each final m_flow_nominal=m1_flow_nominal,
     each dpFixed_nominal=dp1_nominal,
     each final show_T=show_T,
+    each final kFixed=m1_flow_nominal/sqrt(dp1_nominal),
```

Comment:
Please remove that binding (also for `val2`). This is redundant with the binding for `dpFixed_nominal`. There seems to be an inconsistency in the valve model parameters, see https://github.com/ibpsa/modelica-ibpsa/issues/1472. (Sorry I just realize that now.)

Reply:
Removed for `val1` and `val2`.